.gitignore

```diff
@@ -10,4 +10,6 @@ zkevm-metrics
 .vscode
 
 # private envrc file
-.envrc
\ No newline at end of file
+.envrc
+
+ere-guests/**/Cargo.lock
```

Comment:
I think is a good idea to avoid guest programs `Cargo.lock` being interesting for this repo git?